pmlc/dialect/stdx/transforms/BUILD

```diff
@@ -23,6 +23,7 @@ plaidml_cc_library(
     name = "transforms",
     srcs = [
         "boundscheck.cc",
+        "i1StorageToi8.cc",
```

Comment:
Currently, all our filenames are `snake_case`, can you name this `i1_to_i8.cc` for consistency?